packages/select/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "../../tsconfig.json",
+  "include": ["src/**/*", "stories/**/*"]                        /* Avoid stories dir of being compiled */
```

Comment:
Could you please add `"stories/**/*"` to xc-front-kit\packages\popper\tsconfig.json?

Reply:
You can do it yourself :)